src/lib/salad/bloom.c

```diff
@@ -39,29 +39,18 @@
 #include <assert.h>
 #include <string.h>
 
-int
-bloom_create(struct bloom *bloom, uint32_t number_of_values,
-	     double false_positive_rate)
+void
+bloom_init(struct bloom *bloom, uint32_t number_of_values,
```

Comment:
Why did you rename the function? We usually use `_init` suffix for module initialization functions.